src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_RefReadOnly.cs

```diff
@@ -29,7 +32,19 @@ class Test
 }
 ";
 
-            CompileAndVerify(text, verify: Verification.Fails, symbolValidator: module =>
+            void validatePE(PEAssembly assembly)
+            {
+                var reader = assembly.GetMetadataReader();
+                var test = reader.GetTypeDefinitionByName("Test");
+                var method = reader.GetMethodDefinitionByName(test, "M");
+                var returnType = reader.GetParameter(method.GetParameters().ElementAt(1));
```

Comment:
`var returnType = ...ElementAt(0);`?

Reply:
Good catch. Thanks

---
In reply to: [168014179](https://github.com/dotnet/roslyn/pull/24492#discussion_r168014179) [](ancestors = 168014179)